tcks/resteasy-reactive/pom.xml

```diff
@@ -17,7 +17,7 @@
     <properties>
 
         <!-- to avoid sudden surprises, checkout is pinned to a specific commit -->
-        <resteasy-reactive-testsuite.repo.ref>a43f9213d9ebbb239d68c6b7fd4797297e6400bb</resteasy-reactive-testsuite.repo.ref>
+        <resteasy-reactive-testsuite.repo.ref>67e8e59456a5c2cdf3370f7ed38f7f6b576177b9</resteasy-reactive-testsuite.repo.ref>
```

Comment:
Disables a TCK test that ensures that a custom `WriterInterceptor` for `File` is being invoked.
This is very unlikely to be used in the real world and it any case we can't really support it when delegating to Vert.x for the file handling

Reply:
is it not possible to use tags as ref here?

Reply:
It is, but we had decided to be very explicit about this